src/Tools/ExternalAccess/FSharp/Editor/FSharpContentTypeLanguageService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Composition;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.FSharp.Editor
+{
+    [ExportContentTypeLanguageService(LanguageNames.FSharp, LanguageNames.FSharp), Shared]
```

Comment:
💡 Consider using named arguments so we can see what the two `LanguageNames.FSharp` are used for.

Reply:
Makes sense.